docs/kubernetes.md

```diff
@@ -12,7 +12,10 @@ Here are the steps to deploy a simple Kubernetes cluster:
 4. edit the [Kubernetes example](../examples/kubernetes.json) and fill in the blank strings
 5. [generate the template](acsengine.md#generating-a-template)
 6. [deploy the output azuredeploy.json and azuredeploy.parameters.json](../README.md#deployment-usage)
-7. Temporary workaround when deploying a cluster in a custom VNET with Kubernetes 1.5.3:
+  * To enable the optional network policy enforcement using calico, you have to

```

Comment:
The latest push contains the section in `clusterdefinition.md`: https://github.com/Azure/acs-engine/pull/151/files#diff-86820358fc5925785509e38c21c747f1R27